File: Unidad3/Java/u3p1.java
package com.mycompany.Programa1_U3;

import java.util.Scanner;

public class u3p1 {

    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        int edad;
        System.out.println("escribe tu edad ");
        edad =sc.nextInt();
        if (edad>=18)
        System.out.println("Eres mayor de edad.");
        else
        System.out.println("Eres menor de edad.");
    }
}

File: Unidad3/C#/u3p1.cs
using System;

namespace u3p1
{
    internal class program
    {
        static void Main(string[] args)
        {
            int edad;
            Console.WriteLine("¿Qué edad tienes?");
            edad = Convert.ToInt32(Console.ReadLine());
            if (edad >= 18)
                Console.WriteLine("Eres mayor de edad");
            else
                Console.WriteLine("Eres menor de edad");

        }
    }
}